cloud/aws/terraform/common/bucket-notif.bash

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+echo "Running script: ./$(basename $0) $1 $2"
```

Comment:
this is scripts runs as a Terragrunt hook and enables EventBridge notifications on the source bucket. We had to use a hook here because using [aws_s3_bucket_notifiction](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/s3_bucket_notification) deletes any pre-existing notification configuration on the bucket which is annoying as we don't own the bucket. On the other hand, this script has the side effect of enabling eventbridge notifications on the bucket, even after the stack is destroyed. This is usually not an issue as this system will be mostly passive as long as nobody is listening on the other side of the default event bus.